### DISCUSSION TITLE
天啊！才知道有这个邦布充电站

### DISCUSSION THREAD

**Discussion by Jason555647** [share121/inter-knot#5294] [绝区零]
![20240922124247](https://github.com/user-attachments/assets/e1f58090-5e06-4ba5-a3b7-77b94903579c)
好像就我不知道😥

**Comment by Aha-follower** [comment]
停车场那个吗，我记得对着拍照有成就

**Reply by Jason555647** [reply]
是的，确实有

**Comment by wshtdgo** [comment]
没事，我也不知道

**Reply by Jason555647** [reply]
☺️
